state_of_html.yml

```diff
@@ -0,0 +1,28 @@
+locale: pt-BR
+namespace: css
+translations:
+  ###########################################################################
+  # General
+  ###########################################################################
+  # Intro for stateofhtml.com
+  - key: general.state_of_html.intro
+    # aliasFor: general.html2023.survey_intro # Can we do that?
+    t: |
+      Embora os desenvolvedores web tendam a se concentrar em JavaScript e CSS, nada do que fazemos seria possível sem o HTML atuando como base.
+
+      Durante muito tempo parecia que o HTML não estava evoluindo, mas as coisas podem estar mudando.
+      Novos elementos como `<selectlist>` estão no horizonte, novos recursos interessantes como popovers, e uma série de APIs do navegador relacionadas (Web Components, PWAs, etc.).
+
+      Este ano, [Lea Verou](https://lea.verou.me/) assumiu a formidável tarefa de liderar o design desta nova pesquisa, desde o conteúdo à UX e muito mais.
+
+      Com base no sucesso do [Estado do JS](http://stateofjs.com/) e do [Estado do CSS](https://stateofcss.com/), apresentamos o **Estado do HTML**;
+      a última peça que faltava para completar a trilogia, para que possamos acompanhar a evolução da plataforma web como um todo.
+
+      Benefícios para você:
+      - Os resultados da pesquisa são **usados por navegadores** e grupos de padrões **para priorização de roteiros**.
+      Suas respostas podem ajudar a implementar recursos de seu interesse, priorizar incompatibilidades de navegador e resolver lacunas na plataforma.
+      - Aprenda sobre recursos novos e futuros; adicione recursos à sua lista de leitura e obtenha uma lista de recursos no final!
```

Comment:
And this sentence would be: "Aprenda sobre funcionalidades novas e futuras; adicione funcionalidades à sua lista de leitura e obtenha uma lista de recursos no final!"